### PR TITLE
Added Converter chaining and null checker

### PR DESCRIPTION
**Pull request by rbnswartz**

### CODE REVIEW

File: CRMImporter/FieldMap.cs [hunk 1]
```diff
@@ -10,15 +10,15 @@ public class FieldMap
...
         public string TargetField;
-        public IConverter Convert;
+        public IConverter[] Convert;
 
         /// <summary>
         /// FieldMap constructor
         /// </summary>
         /// <param name="source">The field in your source data to match against</param>
         /// <param name="target">The field in CRM to match to</param>
         /// <param name="converter">Optional conversion function</param>
```
Comment (PurpleGuitar) [inline CRMImporter/FieldMap.cs]: Recommend you update comment to match new array parm

File: CRMImporter/FieldMap.cs [hunk 2]
```diff
@@ -10,15 +10,15 @@ public class FieldMap
...
-        public FieldMap(string source, string target, IConverter converter = null)
+        public FieldMap(string source, string target, params IConverter[] converter)
```
Comment (PurpleGuitar) [inline CRMImporter/FieldMap.cs]: Maybe default to empty list so that you don't have to specify a converter?  Looks like if you pass in null it will break `ImportMap.cs:142`.

File: CRMImporterTests/Converters/NullToStringTests.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        public void SetUp()
+        {
+            this.context = new XrmFakedContext();
+            this.service = this.context.GetOrganizationService();
+        }
+
+        [TestMethod]
+        public void TestConvert()
+        {
+            NullToString test = new NullToString();
```
Comment (PurpleGuitar) [inline CRMImporterTests/Converters/NullToStringTests.cs:29]: I recommend adding a test for a custom default value, e.g. `new NullToString("cheese");`

File: CRMImporterTests/ImportMapTests.cs [hunk 4]
```diff
@@ -130,6 +130,16 @@ public void TestWithConverter()
             Assert.AreEqual(true, ImportMap.ConvertValue( trueValue, map, new BooleanAttributeMetadata(), service));
         }
 
+        [TestMethod]
+        public void TestWithMultipleConverters()
+        {
+            string trueValue = "Yes";
+            StringMapTranslation stringMapConverter = new StringMapTranslation(new Dictionary<string, string> { ["Yes"] = "Y" });
+            StringToBool toBoolConverter = new StringToBool("Y");
+            FieldMap map = new FieldMap("source", "dest", stringMapConverter, toBoolConverter);
```
Comment (PurpleGuitar) [inline CRMImporterTests/ImportMapTests.cs:153]: I recommend you add a test for zero converters, and null converter.

File: CRMImporter/FieldMap.cs (hunk 1 above)
Comment (rbnswartz) [reply CRMImporter/FieldMap.cs]: Done.

File: CRMImporterTests/Converters/NullToStringTests.cs (hunk 3 above)
Comment (rbnswartz) [reply CRMImporterTests/Converters/NullToStringTests.cs:29]: There is already one on line 27 although it uses literal " " as the custom value

File: CRMImporterTests/ImportMapTests.cs (hunk 4 above)
Comment (rbnswartz) [reply CRMImporterTests/ImportMapTests.cs:153]: Done.

File: CRMImporter/FieldMap.cs (hunk 2 above)
Comment (rbnswartz) [reply CRMImporter/FieldMap.cs]: The way that params work is that if it isn't specified then it will be an empty array. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params
